core/domain/story_domain_test.py

```diff
@@ -495,6 +501,20 @@ def test_get_acquired_skill_ids_for_node_ids_overlapping_skills(
             ['skill_1', 'skill_2']
         )
 
+    def test_get_acquired_skill_ids_for_node_ids_not_found(
```

Comment:
Please name tests like: `test_{{action}}_with_{{with_condition}}_{{has_expected_outcome}}`

For more details: https://github.com/oppia/oppia/wiki/Backend-tests#guidelines-for-writing-good-tests

ditto below

Reply:
done